Source/items.cpp

```diff
@@ -4994,8 +4977,10 @@ void SpawnBoy(int lvl)
 			GetItemBonus(0, itype, lvl, 2 * lvl, true, true);
 
 			if (!gbIsHellfire) {
-				if (items[0]._iIvalue > 140000)
+				if (items[0]._iIvalue > 140000) {

```

Comment:
Shoudn't this be corrected also?
```suggestion
				if (items[0]._iIvalue > 90000) {
```

Reply:
yes but I left it for another PR as it requires going through this function and figuring out if it matches the original :D and I wasn't feeling like it atm ;p